src/store/modules/common.js

```diff
@@ -44,8 +44,9 @@ const mutations = {
 const actions = {
   // get user info
   getUrlMeta({ commit, state }, { url }) {
+    const ignoreCache = url.includes('_meta_cache')
     const meta = state.metaMap[url]
-    if (meta) {
+    if (meta && !ignoreCache) {
       return new Promise((resolve, reject) => {
         resolve(meta)
       })
```

Comment:
The following is the detailed analysis of the mutations and actions:

* The difference lies in the `getUrlMeta` mutation. It appears that two instances have been defined, which leads to confusion and potential errors when accessing or using these methods.

* As per current guidelines, it's generally recommended not to use the `Promise.then()` method in Vue.js to return promises directly as it violates a component lifecycle rule.
  
  To avoid this violation in production, you could wrap the promise inside another promise object with an `.catch()` to handle possible errors.
- ```javascript
+ import { AxiosResponse } from 'axios';
   ...
```